src/org/labkey/test/TestFileUtils.java

```diff
@@ -635,6 +635,10 @@ private static List<File> unTar(final File inputFile, final File outputDir) thro
             while ((entry = inputStream.getNextEntry()) != null)
             {
                 final File outputFile = new File(outputDir, entry.getName());
+
+                if (!outputFile.toPath().normalize().startsWith(outputDir.toPath()))
```

Comment:
This should prevent the zip slip concern, but probably best to stash `outputDir.toPath().normalize()` outside the loop and use that.